prediction/src/tests/test_endpoints.py

```diff
@@ -48,10 +51,22 @@ def test_endpoint_documentation(client):
     r = client.get(url)
     data = get_data(r)
 
-    assert 'description' in data
+    assert data['description'] == identify.trained_model.predict.__doc__
+
+    url = client.url_for('predict', algorithm='segment')
+    r = client.get(url)
+    data = get_data(r)
+
+    assert data['description'] == segment.trained_model.predict.__doc__
+
+    url = client.url_for('predict', algorithm='classify')
```

Comment:
Feels like we could at least put this into a loop or a test helper instead of duplicating the code twice.